services/net/content/Config/ContentOptions.cs

```diff
@@ -39,7 +39,7 @@ public class ContentOptions : ServiceOptions
     /// <summary>
     /// get/set - Settings around content coming from the Content Migration service
     /// </summary>
-    public ContentMigrationOptions MigrationOptions { get; set; } = new ContentMigrationOptions();
+    public ContentMigrationOptions? MigrationOptions { get; set; } = new ContentMigrationOptions();
```

Comment:
I removed the `?` because it's not needed, nor is it good practice when we can ensure the object is not null ever.